prov/efa/src/rdm/efa_rdm_ep_progress.c

```diff
@@ -600,7 +601,13 @@ ssize_t efa_rdm_ep_send_queued_pkts(struct efa_rdm_ep *ep,
 
 			return ret;
 		}
+
+		pkt_entry->flags &= ~RXR_PKT_ENTRY_RNR_RETRANSMIT;
+		peer = efa_rdm_ep_get_peer(ep, pkt_entry->addr);
+		assert(peer);
+		peer->rnr_queued_pkt_cnt--;
 	}
+
```

Comment:
is it possible to have a unit test for this bug fix? you can mock a `rxr_pkt_entry_sendv` so you don't actually send

Reply:
sure. Let me add it.